theme/scripts/logo.ts

```diff
@@ -7,19 +9,37 @@ export default (function() {
   const Logo = {
     SELECTORS: {
       logoText: "[data-logo-text]",
+      logoTextMobile: "[data-logo-text-mobile]",
     },
     CONSTANTS: {
       index: 'index'
     },
+    STATE: {
+      setInterval: null
+    },
```

Comment:
Let's rename this to `setIntevalId` since `setInterval` returns an id which can be passed to clear interval to clear it.

Reply:
OMG! I actually did this before seeing your comment :-O